dmapiclient/data.py

```diff
@@ -563,15 +568,16 @@ def get_brief(self, brief_id):
         return self._get(
             "/briefs/{}".format(brief_id))
 
-    def find_briefs(self, user_id=None, status=None, framework=None, lot=None, page=None, human=None):
+    def find_briefs(self, user_id=None, status=None, framework=None, lot=None, page=None, human=None, with_users=None):
         return self._get(
             "/briefs",
             params={"user_id": user_id,
                     "framework": framework,
                     "lot": lot,
                     "status": status,
                     "page": page,
-                    "human": human
+                    "human": human,
+                    "with_users": with_users
```

Comment:
More importantly, this is what my API is getting as a request when I run this:

```
http://localhost:5000/briefs?with_users=True
```

So we're sending back `'True'`, meaning we fail the equality test in the API, (which is looking for `'true'`)[https://github.com/alphagov/digitalmarketplace-api/blob/b8bb54b044daf663a9a6401a344fc0cc6c451e8c/app/main/views/briefs.py#L119].